queries.sql

```diff
@@ -1,3 +1,36 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+//Find all animals whose name ends in "mon".
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+
+//List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+
+//List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered='True' AND escape_attempts<3;
```

Comment:
- [ ] Here we have to select name.

`SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';`

Reply:
Hello Deepakdanger,
This very one nothing display after I ran the command given above. What should be done please?
![No output](https://user-images.githubusercontent.com/101729024/205951899-516fa405-1464-4f80-b754-566659ed1980.png)